cloudigrade/account/models.py

```diff
@@ -0,0 +1,15 @@
+"""Cloudigrade Account Models."""
+from django.db import models
+
+from util.models import BaseModel
+
+
+class Account(BaseModel):
+    """Account model."""
+
+    account_id = models.DecimalField(
```

Comment:
`BigIntegerField` instead? Then we don't need the first two kwargs.

Reply:
On the other hand, Django automatically creates an `id` field for all models and this explicit field is redundant. Let's just drop it unless you have another reason I don't see.

Reply:
@ilyavw and I discussed offline… It makes sense to keep this as a 12-digit decimal because that's how AWS defines its account ID: a 12-digit integer.